infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/SwaggerBuilder.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.restapi;
+
+import io.javalin.Javalin;
+import io.javalin.core.JavalinConfig;
+import io.javalin.http.staticfiles.Location;
+import java.util.Optional;
+import java.util.Set;
+import javax.servlet.http.HttpServletRequest;
+import kotlin.jvm.functions.Function1;
+import tech.pegasys.teku.infrastructure.restapi.openapi.OpenApiDocBuilder;
+
+public class SwaggerBuilder {
+  private static final String SWAGGER_VERSION = "4.10.3";
+  private static final Set<String> MODIFIED_FILES =
+      Set.of("/swagger-ui/swagger-initializer.js", "/swagger-ui/swagger-initializer.js.gz");
```

Comment:
my hope is we can use gradle to include a swagger and then just reference that version somewhere like here... This looks like we'll be downloading swagger on load?

Reply:
We are included precompiled swagger from webjars. We need version just for the path in jar

Reply:
It' the only dir in the folder, so I think I could avoid it for sure

Reply:
Looking at the way we could get it and assuming cache issues sometimes, I'd better step back to hardcoded version:
https://github.com/ConsenSys/teku/pull/6012/commits/82f389b53bd6c0db3042b85a137910df0c3b59aa

Reply:
we have a swagger based on the javalin open api plugin. once we remove that, we'll only have our own swagger, and it'll need to be maintained in versions.gradle...
While we continue to use the swagger from javalin, we need to continue to ship both, its very version dependent..

Reply:
In plugin it's hardcoded and I think it's a safer way. Not a very big deal to update one text constant along with gradle version bump. I'd vote for this option